css/style.css

```diff
@@ -1,36 +1,43 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap');
+
+:root {
+  --wrapper-width: 400px;
+}
 
 body {
   min-height: 100vh;
   margin: 0;
   background: #191765;
-
   color: #c8c8d4;
-  font-family: Roboto, 'sans serif';
+  font-family: 'Roboto', sans-serif;
   line-height: 24px;
-  font-size: 15;
+  font-size: 15px;
 }
+
 img {
   position: absolute;
   width: 100%;
   height: 100%;
-  z-index: -1;
 }
+
 .wrapper {
   display: flex;
   flex-direction: column;
 }
+
 .top-container {
   display: flex;
   flex-direction: column;
   padding: 20px 40px;
   background-image: url('/images/headline_section_bg.png');
   background-repeat: no-repeat;
   background-size: cover;
-  height: 100vh;
+  background-position: center;
```

Comment:
- [ ] **REQUIRED:** It will be nice to have the main container of the projects has a display:grid

Reply:
But .top-container is not the main container so in this case, I will make .wrapper display: grid